entries/a/anonaddy.com.json

```
@@ -0,0 +1,13 @@
+{ 
+  
+  "Anonaddy": {
+    "domain": "anonaddy.com",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
```

As long as their _Help Centre_ is not up-to-date, the _Login protection_ paragraph on this page is better than nothing, since it lists the supported options at least.
```suggestion
    ],
    "documentation": "https://anonaddy.com/security/",
```